projects/control-service/projects/pipelines_control_service/build.gradle

```diff
@@ -42,6 +42,9 @@ dependencies { // Implementation dependencies are found on compile classpath of
     implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
     implementation 'org.springframework.retry:spring-retry'
 
+    // Hashicorp Vault client libraries
+    implementation 'org.springframework.vault:spring-vault-core:2.3.3'
```

Comment:
move the version to the versions-of-external-dependencies.gradle

Reply:
done.